packs/bit-bucket/requirements.txt

```diff
@@ -0,0 +1 @@
+bitbucket-api-0.5.0
```

Comment:
This is not a valid Python version specific. Should be something like that: `bitbucket-api>=0.5.0,<0.6`